website/components/Home/CallToAction/CallToAction.module.css

```diff
@@ -0,0 +1,43 @@
+.button {
+    display: inline-block;
+    text-transform: uppercase;
+    cursor: pointer;
+    border-radius: 3px;
+    color: #fefefe;
+    background-color: var(--ifm-color-primary);
+}
+
+.button.large {
+    margin-top: 1rem;
+    padding: 1rem 4rem;
+    text-align: center;
+    font-size: 1rem;
+    font-weight: 500;
+}
+
+.button.outlined.primary {
+    color: var(--ifm-color-primary);
+    border-color: .2rem solid var(--ifm-color-primary);
+}
+
+.button.outlined {
+    background: none;
+    border: .2rem solid #fefefe;
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    margin-bottom: 2rem;
+}
+
+.questions {
+    background-color: #eee;
+    padding: 5rem 0;
+    margin: 0 0;
+}
+
+.questions h6 {
+    margin: -1em 0 1em 0;
+}
```

Comment:
Add a trailing newline at the end of files. As the amount of CSS in here is growing, maybe we should configure [`stylelint`](https://github.com/stylelint/stylelint)?